lib/graph_ql/query_type_creator.rb

```diff
@@ -1,28 +0,0 @@
-require "graphql"
-
-module GraphQL
-  class QueryTypeCreator
```

Comment:
Since we're removing this class, I think we can also remove all the `graphql_` methods in the `Graphqlable` concern and in the `ActsAsTaggableOn` initializer.

Reply:
I think the `Graphqlable` concern should be removed completely, but out of scope for this PR.
I have no idea why `public_created_at` was created...

Reply:
[63f78d3](https://github.com/consul/consul/pull/4766/commits/63f78d30aad8e37137f88c7878e77e305fdf31b7)

Reply:
> I have no idea why public_created_at was created...

That makes two of us :sweat_smile:. No idea why it's important that records return the creation date only showing the hour but not the minute or the second when they were created.